### PR TITLE
Update schema tests to be consistent

### PR DESCRIPTION
**Pull request by doomspork**
@tajchumber can you take a peek at this? I wanted to get ahead of inconsistencies in the code before this project grows too much. In future PRs these are things we should keep an eye out for.

### CODE REVIEW

File: test/companies/schema/company_test.exs [hunk 1]
```diff
@@ -1,12 +1,10 @@
 defmodule Companies.Schema.CompanyTest do
   @moduledoc """
-    This module holds the unit tests to the company changeset
+  This module holds the unit tests to the company changeset
   """
   use Companies.DataCase
 
-  alias Companies.{
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: We don't use the multiple format, or multiline, with only a single module.

File: test/companies/schema/company_test.exs [hunk 2]
```diff
@@ -1,12 +1,10 @@
 defmodule Companies.Schema.CompanyTest do
   @moduledoc """
-    This module holds the unit tests to the company changeset
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: Extra whitespace in the module docs.

File: test/companies/schema/company_test.exs [hunk 3]
```diff
@@ -18,13 +16,13 @@ defmodule Companies.Schema.CompanyTest do
     industry_id: 12346
   }
 
-  describe "company changeset" do
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: The describe block should be the function with its arity.

File: test/companies/schema/company_test.exs [hunk 4]
```diff
@@ -18,13 +16,13 @@ defmodule Companies.Schema.CompanyTest do
...
-    test "company/2 validates with the correct parameters" do
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: `company/2` isn't a function.

File: test/companies/schema/company_test.exs [hunk 5]
```diff
@@ -18,13 +16,13 @@ defmodule Companies.Schema.CompanyTest do
...
+  describe "changeset/2" do
+    test "returns a valid changeset for correct parameters" do
       changeset = Company.changeset(%Company{}, @valid_company_params)
       assert changeset.valid?
     end
 
-    test "errors out if given invalid params" do
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: It's not always possible but let's try to keep the phrasing similar been identical tests.

File: test/companies/schema/industry_test.exs [hunk 6]
```diff
@@ -1,24 +1,22 @@
...
+  This module holds the unit tests to the industry changeset
   """
   use Companies.DataCase
 
-  alias Companies.{
-    Schema.Industry
-  }
+  alias Companies.Schema.Industry
 
-  @valid_industry %{
```
Comment (doomspork) [inline test/companies/schema/industry_test.exs]: Some of these attributes used `_params` and others didn't, I made them consistent.

File: test/companies/schema/job_test.exs [hunk 7]
```diff
@@ -13,14 +13,13 @@ defmodule Companies.Schema.JobTest do
     company_id: 60
   }
 
-  describe "job changesets" do
-    test "validates correct parameters" do
+  describe "changeset/2" do
+    test "returns a valid changeset with correct parameters" do
       changeset = Job.changeset(%Job{}, @valid_job)
       assert changeset.valid?
-      assert changeset.changes.company_id == @valid_job.company_id
```
Comment (doomspork) [inline test/companies/schema/job_test.exs]: This isn't actually testing anything.

File: test/companies/schema/pending_changes_test.exs [hunk 8]
```diff
@@ -18,20 +16,19 @@ defmodule Companies.Schema.PendingChangeTest do
...
   }
 
-  describe "pending_change changesets" do
-    test "validates correct parameters" do
-      changeset = PendingChange.changeset(%PendingChange{}, @valid_pending_change)
+  describe "changeset/2" do
+    test "returns a valid changeset with correct parameters" do
+      changeset = PendingChange.changeset(%PendingChange{}, @valid_pending_change_params)
       assert changeset.valid?
-      assert changeset.changes.user_id == @valid_pending_change.user_id
```
Comment (doomspork) [inline test/companies/schema/pending_changes_test.exs]: Nothing is really being tested here either.

**Comment by zacck** [comment]
@br4inii  @WanjikuMac  hint, hint 💃

**Comment by doomspork** [comment]
@zacck it's no big deal! I really do appreciate your's, @br4inii, and @WanjikuMac's help! Sorry I wasn't able to comment on the PRs sooner, the day got away from me 😁 

Please do continue to contribute, all is appreciated 💜

**Comment by zacck** [comment]
@doomspork  I know they would have wanted to know :)

**Comment by WanjikuMac** [comment]
@zacck is right, @doomspork thanks for sharing the knowledge, much appreciated.

**Pull request merged**